compiler/optimizer/LocalOpts.cpp

```diff
@@ -8131,7 +8132,7 @@ TR_ColdBlockMarker::perform()
    static char *validate = feGetEnv("TR_validateBeforeColdBlockMarker");
    if (validate)
       {
-      TR::ILValidator validator(this);
+      TR::ILValidator validator(comp());
       validator.checkSoundness(comp()->getStartTree());
```

Comment:
This is how it was previously implemented. I'll remove though because it really is redundant.